spec/computer_spec.rb

```diff
@@ -1,8 +1,32 @@
 require "computer"
+require "spec_helper"
```

Comment:
This should be the first require in your specs.

Reply:
Resolved #35